vault/token_store.go

```diff
@@ -3439,10 +3440,16 @@ func (ts *TokenStore) gaugeCollector(ctx context.Context) ([]metricsutil.GaugeLa
 		}
 	})
 
+	if err != nil {
+		return []metricsutil.GaugeLabelValues{}, nil
```

Comment:
If we're deliberately suppressing this error, can we have a comment explaining why?

Reply:
Good catch, thanks. I replaced this to look for the specific error I wanted to suppress (the expiration manager is still in recovery mode) and pass others through.